Formula/s/seaweedfs.rb

```diff
@@ -2,8 +2,8 @@ class Seaweedfs < Formula
   desc "Fast distributed storage system"
   homepage "https://github.com/seaweedfs/seaweedfs"
   url "https://github.com/seaweedfs/seaweedfs.git",
-      tag:      "3.69",
-      revision: "ac6fd36c06f9cd6740cb2ff7e97c1ba92d55bf2c"
+      tag:      "3.70",
```

Comment:
Apparently that version got retracted and tag removed before all checks picked it up:
https://github.com/seaweedfs/seaweedfs/tags
<img width="286" alt="image" src="https://github.com/user-attachments/assets/da92c3ed-89c9-49cb-b78d-8239f52c8b08">